arbitrator/stylus/src/host.rs

```diff
@@ -90,6 +90,36 @@ pub(crate) fn storage_store_bytes32<E: EvmApi>(
     trace!("storage_store_bytes32", env, [key, value], &[])
 }
 
+pub(crate) fn storage_transient_load_bytes32<E: EvmApi>(
+    mut env: WasmEnvMut<E>,
+    key: u32,
+    dest: u32,
+) -> MaybeEscape {
+    let mut env = WasmEnv::start(&mut env, 2 * PTR_INK + EVM_API_INK)?;
+    env.buy_gas(TRANSIENT_BYTES32_GAS)?;
```

Comment:
Would be nice to map errors here to be more informative. To know if it failed at buying gas, at reading, or at transient get

Reply:
the error messages bubbled up are already informative enough